config/optional/core.entity_form_display.node.article.default.yml

```diff
@@ -21,7 +21,6 @@ dependencies:
     - scheduler
     - scheduler_content_moderation_integration
     - select2
-    - thunder
```

Comment:
A config export removes the dependency on thunder. But why? The moderation_state field still uses the thunder_moderation_state_default widget, which is placed in Drupal\thunder\Plugin\Field\FieldWidget